app/src/main/java/ar/scacchipa/jsonplaceholderviewer/ui/MiddleCommentViewModel.kt

```diff
@@ -0,0 +1,22 @@
+package ar.scacchipa.jsonplaceholderviewer.ui
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import ar.scacchipa.jsonplaceholderviewer.data.Comment
+import ar.scacchipa.jsonplaceholderviewer.domain.MiddleCommentsUserCase
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+
+class MiddleCommentViewModel(
+    private val userCase: MiddleCommentsUserCase // = MiddleCommentsUserCase()
+): ViewModel() {
+
+    var middleCommentList = MutableLiveData<List<Comment>?>()
```

Comment:
remember that viewmodels should not expose a mutableLivedata, instead it should be private, and expose to the view a non mutable version of the livedata.

Reply:
the MutableLiveData was encapsulated.